doc/api/stream.md

```diff
@@ -917,6 +917,16 @@ will be ignored.
 Implementors should not override this method, but instead implement
 [`readable._destroy()`][readable-_destroy].
 
+##### readable.readableEncoding
```

Comment:
Needs to be placed after the `##### readable.readable` ABC-wise?

Reply:
Fixed, thanks ~